distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -3,291 +3,320 @@
 
-|federated read
-// add a link: https://github.com/codice/ddf/issues/4374
-|GET
-|The SOURCE ID of a federated source is appended to the URL before the ID of the Metacard to be retrieved is appended to the end.
+.Create Success
+If the ingest is successful, a status of `201 Created` will be returned, along with the Metacard ID in the header of the response.
 
-`<sourceId>` is the `FEDERATED SOURCE ID` and `<metacardId>` is the `Metacard.ID` of the Metacard to be retrieved.
+.Request with Non-XML Data
+[NOTE]
+====
+If a request with non-XML data sent to the Catalog REST endpoint,
+the metacard will be created but the content will stored in the `metadata` field.
+This could affect the discoverability of the resource.
+====
 
-|{url}/sources/&lt;sourceId&gt;/&lt;metacardId&gt;
+If content or metadata is not ingested successfully, check for these error messages.
 
+.Create Error Response Examples
+[cols="2,2,3" options="header"]
+|===
+|Status Code
+|Error Message
+|Possible Causes
 
-|<<{integrating-prefix}catalog_rest_sources_example,sources>>
-|GET
-|Retrieves information about federated sources, including `sourceId`, `availability`, `contentTypes`,and `version`.
+.2+.^|`400 Bad Request`
+.2+.^|`<pre>Error while storing entry in catalog: </pre>`
 
-|{url}/sources/
+|_Malformed XML Response_: If the XML being ingested is not well-formed, an HTTP 400 is returned and the following response body is returned and the specific error is logged in the error log.
+|_Request with Unknown Schema_: If ingest is attempted with a schema that is unknown, unsupported, or not configured by the endpoint, ${branding} creates a generic resource metacard with the provided XML as content for the `metadata` XML field in the metacard.
 
 |===
 
-=== Catalog REST Endpoint Sample Operations
 
-.[[_catalog_rest_create_example]]Create Operation Examples
-The REST endpoint can be used to upload resources as attachments.
-The `create` and `update` methods both support the multipart mime format.
-If only a single attachment exists, it will be interpreted as a resource to be parsed, which will result in a metacard and resource being stored in the system.
+.[[_catalog_rest_read_example]]Read Operation Examples
+The `read` operation can be used to retrieve metadata in different formats.
 
-If multiple attachments exist, then the REST endpoint will assume that 1 attachment is the actual resource (attachment should be named `parse.resource`) and the other attachments are overrides of metacard attributes (attachment names should follow metacard attribute names).
-In the case of the metadata attribute, it is possible to also have the system transform that metadata and use the results of that to override the metacard that would be generated from the resource (attachment should be named `parse.metadata`).
+. Send a `GET` request to the endpoint.
+. Optionally add a `transform` query parameter to the end of the URL with the transformer to be used (such as `transform=kml`).
```

Comment:
Link to available transforms here?

Reply:
I don't know, because that is configurable? I thought about adding the console command to get the list, but I don't think the audience for would necessarily have access to run that command... Maybe a note to check with the admin for available transformers?